### ISSUE TITLE
TypeError: Cannot read property 'start' of undefined

### ISSUE THREAD

**Issue by iahu** [ternjs/tern_for_sublime#185]
```js
{...state, keyN|}
```

Every time when I typing a keyName after a destructuring statement, sublime alert a error message for me.
```
TypeError: Cannot read property 'start' of undefined

                      [ Cancel ]    [Disable Tern]

```

**Comment by othree** [comment]
This is bug belongs to tern itsself

**Comment by othree** [comment]
Should be fixed in 0.24.3
